Library/Formula/dnscrypt-proxy.rb

```diff
@@ -25,6 +25,7 @@ class DnscryptProxy < Formula
   deprecated_option "plugins" => "with-plugins"
 
   depends_on "libsodium"
+  depends_on "minisign"
```

Comment:
I'd probably make this `=> :recommended` so people who aren't interested in using the script can opt-out of the dep.